tfhe/docs/how_to/debug.md

```diff
@@ -0,0 +1,66 @@
+# Debugging FHE code
+
+Since tfhe-rs 0.5, [trivial ciphertexts](./trivial_ciphertext.md) have another application.
+They can be used to allow debugging via a debugger or print statements as well as speeding-up execution time
```

Comment:
what do you mean via a debugger ?

Reply:
If you are the type of people that use debugger, potentially you could do 
```
let value_ = encrypted.try_decrypt_trivial();
```

And inspect `value`, with the debugger

Reply:
ok thanks